account_bank_statement_import_mt940_base/test_files/test-sns-utf8.940

```diff
@@ -0,0 +1,78 @@
+{1:F01SNSBNL2AXXXX0000000000}{2:O940SNSBNL2AXXXXN}{3:}{4:
```

Comment:
Is this real customer data? In that case maybe better anonymize it...

Reply:
@NL66278 I've done nothing more than a copy paste from existing ones in tests. I've added a special character 'à' in it. That's all

Reply:
Maybe we can ask @astirpe who done the first work

Reply:
I got those files from here: https://github.com/OCA/bank-statement-import/pull/125#discussion_r159695587 
Not sure if they are already anonymized.

Reply:
@danger89

Reply:
My bank account was already anonymized. It could be that the other side (which is always a business) is not always anonymized.